recipes/pytest-logging/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest-logging" %}
+{% set version = "2015.11.4" %}
+{% set sha256 = "cec5c85ecf18aab7b2ead5498a31b9f758680ef5a902b9054ab3f2bdbb77c896" %}
+
+package:
+  name: {{ name|lower }}
+  version: "2015.11.4"
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 3
+
+requirements:
+  build:
+    - python
+    - certifi >=2016.9.26
+    - pytest >=2.8.1
+    - setuptools
+
+  run:
+    - python
+    - certifi >=2016.9.26
+    - pytest >=2.8.1
+
+test:
+  imports:
+    - pytest_logging
+
+about:
+  home: https://github.com/saltstack/pytest-logging
+  license: Apache v2.0
```

Comment:
It appears that this package is missing a `Manifest.in` file, so the license isn't in the source on pypi. Please submit an issue in the source repo and put a comment here. I'm not sure that the license file in the Github repo is technically valid since they didn't fill in some stuff, but as a placeholder, you can copy the license into the recipe directory and do something similar to what I did in this other recipe:

https://github.com/conda-forge/staged-recipes/pull/1971/files#diff-13a449f61fa933737908042243c37171R39